notebooks/CondensationBulkAnnotations.R.ipynb

```diff
@@ -100,7 +100,7 @@
    },
    "outputs": [],
    "source": [
-    "server <- OMEROServer(host = 'idr-testing.openmicroscopy.org', username = Sys.getenv('IDR_USER'), password = Sys.getenv('IDR_PASSWORD'), port = as.integer(4064))\n",
+    "server <- OMEROServer(host = Sys.getenv('IDR_HOST'), username = Sys.getenv('IDR_USER'), password = Sys.getenv('IDR_PASSWORD'), port = as.integer(4064))\n",
```

Comment:
The port could be different too
This should be retrieved via system property and if not set fall back to the default port

Reply:
Use `IDR_PORT` if it's set https://github.com/IDR/idr-py/blob/v0.1.1/idr/connections.py#L43

Reply:
Though we may want to switch to `IDR_OMERO_CONFIGURATION_URL` https://github.com/IDR/idr-py/blob/v0.1.1/idr/connections.py#L49